src/main/resources/assets/js/app/inputtype/text/HtmlArea.ts

```diff
@@ -109,14 +109,16 @@ export class HtmlArea
 
         const textAreaWrapper = new DivEl();
 
-        this.editors.push({id: editorId, textAreaWrapper, textAreaEl, property, hasStickyToolbar: false});
-
         textAreaEl.onRendered(() => {
             if (this.authRequest.isFulfilled()) {
-                this.initEditor(editorId, property, textAreaWrapper);
+                this.initEditor(editorId, property, textAreaWrapper).then(() => {
```

Comment:
```
this.initEditor(editorId, property, textAreaWrapper).then(() => {
  this.editors.push({id: editorId, textAreaWrapper, textAreaEl, property, hasStickyToolbar: false});
});
```

This should be extracted into a separate method to avoid code duplication. Actually, do we need a check for `if (this.authRequest.isFulfilled())` at all? If a promise is already resolved, `then` handler will be executed right away.